ONSdigital/rm-actionexporter-service
# Simplify template mapping

# Motivation and Context
Simplify the template mechanism in action exporter. Templates are no longer loaded from the database. Instead they are packaged into the jar file and stored in source control where they can
be reviewed and the history kept. They rarely ('never') change and if we need a new one it is easy to add to source control and deploy a new version now. 

# What has changed
1. Drop the template table
2. Update the template mapping table, remove foreign key constraints, update column names and the enter the full name of the template.
3. Remove the specific Freemarker configuration
4. Remove all the social templates as RASRM only does business surveys now
5. Remove an integration test as it was only testing social surveys

# How to test?
1. Deploy into your kubernetes environment 
2. Schedule a collection exercise to go live
3. Ensure print file is generated (enabling GCS bucket storage allows you to get to the print file)

# Links
https://trello.com/c/hRCBXqVF/2034-s11a-document-action-exporter-1-week
